src/main/java/pl/onsight/wypozyczalnia/validator/impl/InputValidatorImpl.java

```diff
@@ -0,0 +1,20 @@
+package pl.onsight.wypozyczalnia.validator.impl;
+
+import org.springframework.stereotype.Component;
+import pl.onsight.wypozyczalnia.validator.InputValidator;
+
+@Component
+public class InputValidatorImpl implements InputValidator {
+    @Override
+    public boolean isInputValid(String input) {
+
```

Comment:
to na pewno wszystkie znaki? :P mozna by bylo sprobowac blokowac przez nr w ascii, wydaje mi sie ze uproscioloby to troche kod